### PR TITLE
Add _prepare_picking_assign method for easier inheritance

### PR DESCRIPTION
**Pull request by alexis-via**
Inheritance is a key advantage of Odoo, that's why we need this kind of _prepare methods.

### CODE REVIEW

**Comment by clonedagain** [comment]
Could you please explain in the docstring at which moment the method is called? That will make it easier to find out which one we want to override.

**Comment by jco-odoo** [comment]
Looks fine by me.

File: addons/stock/stock.py [hunk 1]
```diff
@@ -2068,6 +2068,17 @@ def onchange_product_id(self, cr, uid, ids, prod_id=False, loc_id=False, loc_des
             result['location_dest_id'] = loc_dest_id
         return {'value': result}
 
+    def _prepare_picking_assign(self, cr, uid, move, context=None):
+        """This method is designed to be inherited"""
```
Comment (jco-odoo) [inline addons/stock/stock.py]: Maybe a better name also?  """Prepares a new picking for this move as it could not be assigned to another picking"""

**Comment by alexis-via** [comment]
OK, could we merge it then ?

Note that Akretion has signed the CLA (the red status is just an issue with the email address is used in this old commit).

**Comment by jco-odoo** [comment]
Merged with 404b668370

**Pull request closed without merging**